cypress/e2e/models/migration/applicationinventory/analysis.ts

```diff
@@ -321,6 +355,32 @@ export class Analysis extends Application {
             });
     }
 
+    public static verifyAllAnalysisStatuses(status) {
+        cy.log(`Verifying all analysis statuses, expecting ${status}`);
+        cy.get(analysisColumn, { log: false }).each(($el) => {
+            cy.wrap($el)
```

Comment:
Here also May be we can just use the existing verifyanaysisstatus() method inside the loop

Reply:
Solved in 8478094201669b508dfc5d989a60e44afad52160.